app/scripts/modules/google/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -176,18 +176,40 @@ module.exports = angular.module('spinnaker.gce.serverGroupCommandBuilder.service
     function populateCustomMetadata(metadataItems, command) {
       // Hide metadata items in the wizard.
       if (metadataItems) {
+        let customUserData = '';
+        let customUserDataKeys = [];
         if (angular.isArray(metadataItems)) {
           metadataItems.forEach(function (metadataItem) {
-            if (!_.includes(gceServerGroupHiddenMetadataKeys, metadataItem.key)) {
+            if (metadataItem.key === 'customUserData') {
```

Comment:
What happens if `{ key: 'customUserData' , value: ... }` is the last element in the `metadataItems` array? Will I have omitted my custom userdata items from `command.instanceMetadata`?